miio/device.py

```diff
@@ -248,45 +290,26 @@ def status(self) -> DeviceStatus:
         """Return device status."""
         raise NotImplementedError()
 
-    def actions(self) -> Dict[str, ActionDescriptor]:
+    def actions(self) -> Optional[Dict[str, ActionDescriptor]]:
```

Comment:
I don't think these should be marked as `Optional`: if the device has no actions defined, we should just have an empty list. This will make the API cleaner as the downstreams do not need to check against noneness but can simply iterate over the list directly.

Reply:
Yes mypy was complaining, will fix it another way.